nodejs/nodejs-guestbook/skaffold.yaml

```diff
@@ -1,43 +1,27 @@
-# To learn more about the skaffold.yaml schema visit
-# https://skaffold.dev/docs/references/yaml/
-apiVersion: skaffold/v2beta4
+# The Guestbook app is configured to use skaffold modules.
+# The frontend and backend services can be activated by name.
+# To learn more about skaffold modules and multi-config support,
+# visit https://skaffold.dev/docs/design/config/
+
+apiVersion: skaffold/v2beta15
 kind: Config
+metadata:
+  name: frontend-svc
 build:
-  tagPolicy:
-    sha256: {}
-  # defines where to find the code at build time and where to push the resulting image
-  artifacts:
-  - image: nodejs-guestbook-backend
-    context: src/backend
-  - image: nodejs-guestbook-frontend
-    context: src/frontend
-# defines the Kubernetes manifests to deploy on each run
-deploy:
-  kubectl:
-    manifests:
-    - ./kubernetes-manifests/*.service.yaml
-    - ./kubernetes-manifests/*.deployment.yaml
-portForward:
-  # Cloud Code automatically port forwards resources,
-  # or users can define explicit port forwards here.
-  # For more info, visit https://skaffold.dev/docs/pipeline-stages/port-forwarding/#UDPF
-  - resourceType: service
-    resourceName: nodejs-guestbook-frontend
-    port: 80
-    localPort: 4503
-profiles:
-# use the cloudbuild profile to build images using Google Cloud Build
-- name: cloudbuild
-  build:
-    googleCloudBuild: {}
-- name: buildpacks
-  build:
-    artifacts:
-    - image: nodejs-guestbook-backend
-      context: src/backend
-      buildpack:
-        builder: "gcr.io/buildpacks/builder:v1"
-    - image: nodejs-guestbook-frontend
-      context: src/frontend
-      buildpack:
-        builder: "gcr.io/buildpacks/builder:v1"
+  local:
+    concurrency: 0
+requires:
```

Comment:
`frontend-svc` requiring `backend-svc` essentially makes running `skaffold dev` identical to running `skaffold dev --module=frontend-svc`. I suggest not having this `requires`. Then:
- running `skaffold dev --module=frontend-svc` would only run `frontend-svc`
- running `skaffold dev --module=backend-svc` wound run only `backend-svc`
- running `skaffold dev` would run both `frontend-svc` and `backend-svc`

Reply:
So we'd be able to test the robustness of the frontend in the face of the backend being unavailable.

Reply:
Thanks for the explanation, I've removed the `backend-svc` requirement.

Reply:
Maybe in the (currently unwritten) modules Neos tutorial using Guestbook, we'll include a step for the user to add the backend-svc as a required config, to demonstrate how they can couple modules together if they need to.